src/fmu/tools/rms/qcreset.py

```diff
@@ -209,3 +208,194 @@ def set_surfaces(
             )
 
     print("End of function set_data_constant().")
+
+
+def set_data_empty(config: Dict):
+    """Set data from RMS empty.
+
+    This method is a utility in order to set empty surface and 3D grid property
+    data. The value must be of the correct type (if discrete 3D property for
+    example). The purpose of it is to make sure that those data are properly
+    generated by the modelling workflow and not inherited from a previous run
+    with the corresponding jobs deactivated.
+
+    The input of this method is a Python dictionary with defined keys. The keys
+    "project" and "value" are required while "horizons", "zones" and
+    "grid_models" are optional (at least one of them should be provided for the
+    method to have any effect).
+
+    Args:
+        project: The roxar magic keyword ``project`` refering to the current
+            RMS project.
+
+        horizons: A Python dictionary where each key corresponds to the name of
+            the horizons category where horizon data need to be made empty. The
+            value associated to this key should be a list of horizon names to
+            modify. If a string ``all`` is assigned instead of a list, all
+            available horizon names for this category will be used.
+            Alternatively, if a list of horizons categories is given instead of
+            a dictionary, the method will apply to all horizons within these
+            horizons categories.
+
+        zones: A Python dictionary where each key corresponds to the name of
+            the zones category where zone data need to be made empty. The value
+            associated to this key should be a list of zone names to modify. If
+            a string ``all`` is assigned instead of a list, all available zone
+            names for this category will be used.
+            Alternatively, if a list of zones categories is given instead of a
+            dictionary, the method will apply to all zones within these zones
+            categories.
+
+        grid_models: A Python dictionary where each key corresponds to the name
+            of the grid models where properties need to be made empty. The
+            value associated to this key should be a list of property names to
+            modify. If a string ``all`` is assigned instead of a list, all
+            available properties for this grid model name will be used.
+            Alternatively, if a list of grid models names is given instead of a
+            dictionary, the method will apply to all properties within these
+            grid models.
+    """
+
+    assert isinstance(config, dict), "Argument must be a Python dictionary!"
```

Comment:
This would raise the exception `AssertionError`, while I believe a `TypeError` would be more correct.